### PR TITLE
Cclient - add read/write method in response/request objects

### LINKED ISSUE #805: cclient: add read/write method in response/request objects.
As a user, response/request objects should provide get/set method to access object members without knowing underlying operations.

### PR DESCRIPTION
**Pull request by oopsmonk**
This PR fixed #805 and #828 

# Test Plan:
CI pass

### CODE REVIEW

File: cclient/request/find_transactions.h [hunk 1]
```diff
@@ -47,21 +47,37 @@ static inline retcode_t find_transactions_req_bundle_add(
...
+static inline flex_trit_t* find_transactions_req_tag_get(
+    find_transactions_req_t* const req, size_t index) {
+  return hash243_queue_at(&req->tags, index);
+}
 
 static inline retcode_t find_transactions_req_approvee_add(
     find_transactions_req_t* const req, flex_trit_t const* const hash) {
   return hash243_queue_push(&req->approvees, hash);
 }
+static inline flex_trit_t* find_transactions_req_approve_get(
```
Comment (tsvisabo) [inline cclient/request/find_transactions.h]: typo: approvee

**Review by tsvisabo — CHANGES_REQUESTED** [review]

File: cclient/response/get_balances.h [hunk 2]
```diff
@@ -17,16 +17,26 @@ extern "C" {
 static const UT_icd ut_uint64_icd UTARRAY_UNUSED = {sizeof(uint64_t), NULL,
                                                     NULL, NULL};
 typedef struct {
-  UT_array* balances;
   int milestoneIndex;
```
Comment (thibault-martinez) [inline cclient/response/get_balances.h]: Not part of this PR but can you switch this to snake_case ?

**Review by thibault-martinez — CHANGES_REQUESTED** [review]

File: cclient/response/get_balances.h (hunk 2 above)
Comment (oopsmonk) [reply cclient/response/get_balances.h]: sure

**Review by thibault-martinez — DISMISSED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Review by tsvisabo — APPROVED** [review]
